temboardagent/plugins/dashboard/db.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+
+import json
+import os
+import sqlite3
+from textwrap import dedent
+
+
+def bootstrap(path, dbname):
+    """Create SQLite3 database model to store stuff."""
+
+    with sqlite3.connect(os.path.join(path, dbname)) as conn:
+        c = conn.cursor()
+        c.execute("DROP TABLE IF EXISTS dashboard_metrics")
+        c.execute(
+            dedent("""
+                CREATE TABLE dashboard_metrics (
```

Comment:
The table could simply be named "metrics" as in monitoring part in previous commit.

Reply:
Make sense. Done.